src/main/kotlin/racingcar/RacingCarGame.kt

```diff
@@ -0,0 +1,28 @@
+package racingcar
+
+class RacingCarGame(numberOfCar: Int, private val numberOfTrial: Int) {
+    private val cars: List<Car> = (0..numberOfCar).map { i -> Car(i) }.toList()
+
+    fun startGame(): RacingCarGameSnapShots {
+        for (i in 0 until numberOfTrial) {
+            move()
+        }
+        return RacingCarGameSnapShots(cars)
+    }
+
+    private fun move() {
+        for (i in cars.indices) {
+            val nextMovement = decideMovement()
+            cars[i].move(nextMovement)
+        }
+    }
+
+    private fun decideMovement(): Int {
+        val random: Int = ((Math.random() * 10)).toInt()
+        if (random < 4) {
+            return 0
+        } else {
+            return random
+        }
```

Comment:
```suggestion
        return if (random < 4) {
            0
        } else {
            random
        }
```
코틀린의 if ~ else 는 [문이 아닌 식](https://blog.leocat.kr/notes/2019/12/29/kotlin-statement-and-expression)이기 때문에 위와 같이 사용할 수 있습니다.